index.html

```diff
@@ -0,0 +1,56 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Zuhra Hashimi</title>
+</head>
+<body>
+    <header>
+        <nav class="topnav">
+            <a href="#" class="logo">Zuhra Hashimi</a>
+            <div class="page_header">
+                <img class="img" src="images/Icon - Menu.png" alt="">
+            </div>
+        </nav>
```

Comment:
[OPTIONAL] Kindly move the logo outside the nav tag into the header tag for semantic reasons 🤗
- _Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info._